src/js/components/project-view/branch-summary.tsx

```diff
@@ -37,27 +37,27 @@ const BranchSummary = (props: Props) => {
     deploymentForLatestCommit,
     latestCommit,
   } = props;
-  let commitContent: JSX.Element;
+  let commitContent: JSX.Element | undefined;
 
   if (!latestSuccessfullyDeployedCommit) {
-    commitContent = (
-      <div className={styles.empty}>
-        No previews available
-      </div>
-    );
+    commitContent = undefined;
```

Comment:
`commitContent` is already `undefined` at this point. You can change the if clause to just 
```
if (latestSuccessfullyDeployedCommit) {
``` 
and remove this code block.